### PR TITLE
[#12538] Instructor sessions page: Empty deleted sessions table is not displayed

### LINKED ISSUE #12538: Instructor sessions page: Empty deleted sessions table is not displayed
**Current Behavior**

The instructor sessions page does not display an empty table of deleted feedback sessions, if there are no deleted sessions.

With deleted sessions:
Screen Shot 2023-07-31 at 4 32 35 PM

No deleted sessions:
Screen Shot 2023-07-31 at 4 32 52 PM

**Expected Behavior**

With reference to #10508:

> I think empty tables should be shown even if there are no deleted/archived items. That's more consistent with our policy that even disabled items are shown to let users know where things are.

**Suggestion**

- Add an alert for Deleted Sessions table when there are no deleted courses (Similar to #12521)

### PR DESCRIPTION
**Pull request by mariemllr**
Fixes #12538

**Outline of Solution**

I added an alert for Deleted Sessions table that is displayed when there are no deleted courses.
Bildschirmfoto 2023-08-05 um 12 05 56

### CODE REVIEW

File: src/web/app/components/sessions-recycle-bin-table/sessions-recycle-bin-table.component.html [hunk 1]
```diff
@@ -91,5 +97,6 @@ <h2 class="text-muted">
         </table>
       </div>
     </div>
+    </div>
   </div>
```
Comment (domlimm) [inline src/web/app/components/sessions-recycle-bin-table/sessions-recycle-bin-table.component.html:101]: Nit on L99: Some alignment issue please check.

Currently:
image

Should be:
image

**Review by domlimm — COMMENTED** [review]
Just a nit. Else, LGTM!

File: src/web/app/components/sessions-recycle-bin-table/sessions-recycle-bin-table.component.html (hunk 1 above)
Comment (mariemllr) [reply src/web/app/components/sessions-recycle-bin-table/sessions-recycle-bin-table.component.html]: Fixed!

**Review by domlimm — APPROVED** [review]
LGTM! Thank you for your contribution!

**Review by weiquu — APPROVED** [review]
LGTM!

**Pull request merged**